CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 - The `-jdk14` and `-jdk15` docker image variants will be removed in a future release. JDK 14 and 15 are no longer receiving security updates from upstream vendors.
   Note that the default docker image usage JDK 16 and is still receiving security updates.
 - The commandline option `--validators-performance-tracking-enabled` has been deprecated in favour of `--validators-performance-tracking-mode`
+- New commandline option `--Xmetrics-endpoint` has been added to allowing Teku to publish metrics to a thirdy party provider (currently supported Beaconcha.in).
```

Comment:
I wouldn't add this to the changelog until we're ready to make it a fully supported option (without the `X` prefix).